src/Http2.re

```diff
@@ -246,97 +261,135 @@ module ServerHttp2Session = {
 };
 
```

Comment:
This module needs to be cleaned up to conform to the standard module structure of stream types. Will do this in a future PR.